install/tools/ipa-replica-manage.in

```diff
@@ -235,19 +239,7 @@ def list_replicas(realm, host, replica, dirman_passwd, verbose, nolookup=False):
 
     for entry in entries:
         print('%s: %s' % (entry.single_value.get('nsds5replicahost'), ent_type))
-
-        if verbose:
-            initstatus = entry.single_value.get('nsds5replicalastinitstatus')
-            if initstatus is not None:
-                print("  last init status: %s" % initstatus)
-                print("  last init ended: %s" % str(
-                    ipautil.parse_generalized_time(
-                        entry.single_value['nsds5replicalastinitend'])))
-            print("  last update status: %s" % entry.single_value.get(
-                'nsds5replicalastupdatestatus'))
-            print("  last update ended: %s" % str(
-                ipautil.parse_generalized_time(
-                    entry.single_value['nsds5replicalastupdateend'])))
+        print_replication_status(entry, ent_type, verbose)
```

Comment:
This differs from the function declaration, ent_type and verbose are swapped.

Reply:
Fixed, thank you.